stubs/simplejson/simplejson/__init__.pyi

```diff
@@ -7,8 +9,83 @@ from simplejson.raw_json import RawJSON as RawJSON
 from simplejson.scanner import JSONDecodeError as JSONDecodeError
 
 _LoadsString: TypeAlias = str | bytes | bytearray
+_T = TypeVar("_T")
 
-def dumps(obj: Any, *args: Any, **kwds: Any) -> str: ...
-def dump(obj: Any, fp: IO[str], *args: Any, **kwds: Any) -> None: ...
-def loads(s: _LoadsString, **kwds: Any) -> Any: ...
-def load(fp: IO[str], **kwds: Any) -> Any: ...
+def dumps(
+    obj: Any,
+    skipkeys: bool = ...,
+    ensure_ascii: bool = ...,
+    check_circular: bool = ...,
+    allow_nan: bool = ...,
+    cls: type[JSONEncoder] | None = ...,
+    indent: str | int | None = ...,
+    separators: tuple[str, str] | None = ...,
+    encoding: str = ...,
+    default: Callable[[Any], Any] | None = ...,
+    use_decimal: bool = ...,
+    namedtuple_as_object: bool = ...,
+    tuple_as_array: bool = ...,
+    bigint_as_string: bool = ...,
+    sort_keys: bool = ...,
+    item_sort_key: Callable[[Any], SupportsRichComparison] | None = ...,
+    for_json: bool = ...,
+    ignore_nan: bool = ...,
+    int_as_string_bitcount: int | None = ...,
+    iterable_as_array: bool = ...,
+    # Technically: you cannot pass extra `kw` without passing custom `cls`
```

Comment:
Might be useful to handle this with overloads. Being able to remove `**kwargs` from the stub is useful because it means type checkers can tell you if you get a parameter name wrong.

Reply:
Done.

Reply:
This comment is no longer necessary now that we have the overloads, correct?
```suggestion
```